src/SymfonyCache/HttpCacheAwareKernelInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\SymfonyCache;
+
+use Symfony\Component\HttpKernel\HttpCache\HttpCache;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+interface HttpCacheAwareKernelInterface extends HttpKernelInterface
```

Comment:
why do we extend the kernel interface here? i think the consuming code does not care what is providing the http cache. this could be `HttpCacheProvider` and the trait `HttpCacheAware`

Reply:
It should still have a `handle()` method and thus be a valid kernel ;-)

Reply:
no, we never call handle on this, we only call getHttpCache

Reply:
That's right, removed in ca31460557d3d63577048b82fb632c3cf98ecac8.